CHANGES.rst

```diff
@@ -18,6 +18,40 @@ The current role maintainer_ is drybjed_.
 
 .. _debops.ifupdown master: https://github.com/debops/ansible-ifupdown/compare/v0.3.2...master
 
+Added
+~~~~~
+
+- Add a separate ``iface@.service`` :command:`systemd` unit file which is used
+  to setup network interfaces that don't have an explicit device related to
+  them (bridges, VLAN interfaces, bonding interfaces).
+
+  This is required because of the recent changes in the ``ifupdown`` package in
+  Debian Jessie/Stretch which change the ``ifup@.service`` unit file to have
+  the related device of a given interface as a dependency. As a result, the
+  ``ifup@.service`` instances that don't have an explicit device associated
+  with them (anything other than a physical Ethernet network interface) wait
+  endlessly for a device that it's not present. This doesn't happen if the
+  network interfaces are configured using :command:`systemd-networkd`
+  facilities, but because this role provides a pure ``ifupdown``
+  implementation, this was the easiest way to solve that issue. [drybjed_]
+
+- Add support for the ``weight_class`` interface parameter which allows to
+  affect filename sorting. [drybjed_]
+
+Changed
+~~~~~~~
+
+- Change the weight numbers from 2 to 3 digits. This might affect network
```

Comment:
This is mostly a cosmetic change. I plan to use the 3-digit version in the new `debops.ferm` configuration to allow for higher range of rule weights (000-999 instad of 00-99). Since I ported these changes from `debops.ferm` here, it felt natural to switch to similar system, although I don't think that the higher range would be needed for the interfaces. I can switch to the 00-00 range pretty easily, do you think that it should be done?